area_corners:
- - - 49.9
    - 8.9
  - - 49.90005
    - 8.9
  - - 49.90005
    - 8.89995
  - - 49.9
    - 8.89995
  - 3
initial_waypoints:
- - - 49.9
    - 8.9
    - 2.0
search_area_number: 1

